aspnetcore/web-api/advanced/formatting.md

```diff
@@ -72,7 +72,7 @@ Content *negotiation* takes place when an `Accept` header appears in the request
 
 If no formatter is found that can satisfy the client's request, ASP.NET Core:
 
-* Returns `406 Not Acceptable` if <xref:Microsoft.AspNetCore.Mvc.MvcOptions> has been set, or -
+* Returns `406 Not Acceptable` if `ReturnHttpNotAcceptable` is set to true on <xref:Microsoft.AspNetCore.Mvc.MvcOptions>, or -
```

Comment:
```suggestion
* Returns `406 Not Acceptable` if <xref:Microsoft.AspNetCore.Mvc.MvcOptions.ReturnHttpNotAcceptable?displayProperty=nameWithType> is set to `true`, or -
```

@Rick-Anderson ... Here's the API cross-link if you want it. I'm just sitting here wrapping things up today and waiting for an engineer to get back to me, so I thought I'd post this for you if you want it.